scripts/e2e-tests.sh

```diff
@@ -269,6 +268,11 @@ function setup_test_cluster() {
 
   header "Setting up test cluster"
 
+  # Set the actual project the test cluster resides in
+  # It will be a project assigned by Boskos if test is running on Prow, 
+  # otherwise will be ${GCP_PROJECT} set up by user.
+  export E2E_CLUSTER_PROJECT="$(gcloud config get-value project)"
```

Comment:
can we call this E2E_PROJECT_ID or something like that? cluster_project seems to inidcate it is a property of the cluster, which this is not as this can potentially be used for any non-cluster operation as well.

Reply:
There are other vars like `E2E_CLUSTER_ZONE`, `E2E_CLUSTER_REGION`, `E2E_CLUSTER_NAME`, etc, that's why I pick up this name to follow the convention.
@adrcunha Do you have any suggestions for it?

Reply:
I agree with E2E_PROJECT_ID, as it's in sync with PROJECT_ID.

Reply:
Fixed.